.github/ISSUE_TEMPLATE.md

```diff
@@ -10,7 +10,9 @@ Note that leaving sections blank will make it difficult for us to troubleshoot a
 
 **What version of ESLint are you using?**
 
-**What configuration and parser (Espree, Babel-ESLint, etc.) are you using?**
+**What parser (Espree, Babel-ESLint, etc.) are you using?**
```

Comment:
I'd maybe phrase it as, "What parser (default, Babel-ESLint, etc.) are you using?" The word "Espree" has little meaning to most ESLint users.